### DISCUSSION TITLE
NFC for iOS?

### DISCUSSION THREAD

**Discussion by Meffesino** [donetick/donetick#40] [Ideas]
Hi, I would love to use NFC with my iOS device. Unfortunately it seems like there is no NFC support for browsers via iOS. 
Any work-around or plans to support NFC for iOS?

**Comment by meauxt** [comment]
@Meffesino Hi there! Unfortunately, the only workaround for now is to generate a URL and use another app to write the URL for the NFC tag. Will have a way to do that in the next release

**Reply by FracKenA** [reply]
Any update on this?

**Reply by meauxt** [reply]
@Meffesino  @FracKenA, this is completed! Check out the iOS app!
